src/index.js

```diff
@@ -0,0 +1,80 @@
+import './style.css';
+import { countShows } from './modules/counterItem.js';
+import createShowCard from './modules/display.js';
+
+const API = 'https://api.tvmaze.com/shows/1/episodes';
+const likesAPI = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/OpXkQhwbfD4wnLSWy6wV/likes';
+
+const displayEpisode = document.querySelector('.episode');
+const previous = document.querySelector('.previous');
+const next = document.querySelector('.next');
+const pageNum = document.querySelector('.page-numbers');
+const showsHeader = document.querySelector('.shows-header');
+const body = document.querySelector('body');
+
+let showsArray = [];
+
+const getData = async (API) => {
+  const result = await fetch(API);
+  return result;
+};
+
+const displayShows = (shows, pageNumber, heading) => {
+  shows.slice(pageNumber * 10 - 10, pageNumber * 10).forEach((show) => {
+    const div = createShowCard(show);
+    displayEpisode.append(div);
+  });
+
+  const showsCount = countShows(showsArray);
+  heading.innerHTML = `Episodes: ${showsCount}`;
+};
+
+function loadNext(pageNumber, shows, showsHeader) {
+  const nextPage = pageNumber + 1;
+  if (nextPage < 5) {
+    pageNum.innerHTML = nextPage;
+    displayEpisode.innerHTML = '';
+    getData(API)
+      .then((response) => response.json())
+      .then((json) => displayShows(json, pageNumber + 1));
```

Comment:
- [ ] Great job so far 👏🏻 👏🏻  However According to the project requirement,  you should Use `async` and `await` with the `fetch` method to send and receive data. You're on the good way but you only need to avoid using Promises and change your syntax.
- Syntax: Follow this syntax which we could use in order to execute an asynchronous method async-await:

```
const submitScore = async (id, user, score) => {
  const response = await fetch(
    `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${id}/scores/`,
    {
      method: 'POST',
      headers: {
        'Content-Type': 'application/json',
        Accept: 'application/json',
      },
      body: JSON.stringify({
        user,
        score,
      }),
    },
  );
  const data = await response.json();
  return data.result;
};
``` 
- For more documentation, you can check  [Promises vs Async Await](https://www.loginradius.com/blog/engineering/callback-vs-promises-vs-async-await/)

* [ ] Note, you should ultimately consider not using the ["callback hell"](https://www.freecodecamp.org/news/how-to-deal-with-nested-callbacks-and-avoid-callback-hell-1bc8dc4a2012/) with a lot of `.then()` methods while using async/await promises. Just like the `.then` method, you can use `await` multiple times.

Reply:
Done.